common.css

```diff
@@ -86,6 +89,27 @@ span{
     display: grid;
     grid-template-columns: 300px auto;
```

Comment:
Is there a reason you used px instead of rem?

Reply:
I will change it to rems